src/os/bluestore/BlueFS.cc

```diff
@@ -53,7 +53,7 @@ BlueFS::~BlueFS()
 
 void BlueFS::_init_logger()
 {
-  PerfCountersBuilder b(cct, "BlueFS",
```

Comment:
commit message is still misleading; this doesn't add any metrics, it just makes them have a nick (and changes the upper-level name to lowercase)